<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TempControlView</class>
 <widget class="QFrame" name="TempControlView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1025</width>
    <height>72</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0,1,0,1,0">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label1_2">
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Temperture：</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="setingtBtn">
     <property name="text">
      <string>Seting</string>
     </property>
     <property name="icon">
      <iconset resource="../mainwindow.qrc">
       <normaloff>:/icons/seting.png</normaloff>:/icons/seting.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>64</width>
       <height>64</height>
      </size>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="starttBtn">
     <property name="text">
      <string>start</string>
     </property>
     <property name="icon">
      <iconset resource="../mainwindow.qrc">
       <normaloff>:/icons/start.png</normaloff>:/icons/start.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>64</width>
       <height>64</height>
      </size>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label1">
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Target Temperture:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLCDNumber" name="targetLcdNumber">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>29</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
     <property name="value" stdset="0">
      <double>25.000000000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label2">
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>℃/Current Temperture :</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLCDNumber" name="currentLcdNumber">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>29</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="smallDecimalPoint">
      <bool>false</bool>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label3">
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>℃</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../mainwindow.qrc"/>
 </resources>
 <connections/>
</ui>
